### PR TITLE
revive eospac

### PR DESCRIPTION
**Pull request by Yurlungur**
Bits have rotted. This is bit necromancy.

### CODE REVIEW

**Comment by Yurlungur** [comment]
@chadmeyer @dholladay00 this is ready for review.

I also do some clang formatting and rename `test_eos_spiner` to `eos_unit_tests` and `mix` to `closure`.

File: eos/eos_eospac.cpp [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+  DPDT = dy[0];
+  DPDE = DPDT / (DEDT + EPS);
+  BMOD = rho_ref_ * DPDR + DPDE * (PRESS / (rho_ref_ + EPS) - rho_ref_ * DEDR);
+  bmod_ref_ = BulkModulusFromSesame(std::max(BMOD, 0.0));
+  dpde_ref_ = PressureFromSesame(SieToSesame(DPDE));
+  dvdt_ref_ = dpde_ref_ * cv_ref_ /
+              (rho_ref_ * rho_ref_ * PressureFromSesame(DPDR) + EPS);
+}
+
+PORTABLE_FUNCTION Real EOSPAC::TemperatureFromDensityInternalEnergy(
```
Comment (dholladay00) [inline eos/eos_eospac.cpp:98]: Are these actually device callable?

Comment (Yurlungur) [reply eos/eos_eospac.cpp:98]: No they're not. And cmake will yell at you if you try. I figured I'd leave it though... for... consistency with the other EOS's I suppose.

Comment (dholladay00) [reply eos/eos_eospac.cpp:98]: Maybe Cmake preventing it is enough, but it looks a little deceptive. If it's well documented that this isn't GPU capable then it's ok.

Comment (Yurlungur) [reply eos/eos_eospac.cpp:98]: It's documented. :+1:

**Comment by dholladay00** [comment]
@Yurlungur this looks good to me. I don’t see an approve button but I approve

**Comment by Yurlungur** [comment]
Cool, thanks @dholladay00 ! I'll merge it in then. (For future reference, the approve button is in the code tab)

**Pull request merged**